catroid/src/org/catrobat/catroid/physics/PhysicsLook.java

```diff
@@ -59,6 +63,7 @@ public void setLookData(LookData lookData) {
 		super.setLookData(lookData);
 		PhysicsWorld physicsWorld = ProjectManager.getInstance().getCurrentProject().getPhysicsWorld();
 		physicsWorld.changeLook(physicsObject, this);
+		updatePhysicsObjectState(true);
```

Comment:
if I remember correctly, that parameter was used for a staterecording (especially for the hangup state, that had to be recorded to restart if the hangup is over)
That whole concept should probably be mentioned during the refactor/extention of the PhysicsObjectStateHandler.